intl.emu

```diff
@@ -142,51 +136,47 @@
       <dl class="header">
         <dt>description</dt>
         <dd>
-          <p><ins>If _x_ is a mathematical value, it</ins> <del>It</del> involves solving the following equation, which returns a valid mathematical value given integer inputs:</p>
+          <p>It involves solving the following equation, which returns a valid mathematical value given integer inputs:</p>
           <emu-eqn id="eqn-ToRawPrecisionFn" aoid="ToRawPrecisionFn">
             ToRawPrecisionFn(_n_, _e_, _p_) = _n_ × 10<sup>_e_–_p_+1</sup>
             where 10<sup>_p_–1</sup> ≤ _n_ < 10<sup>_p_</sup>
           </emu-eqn>
         </dd>
       </dl>
       <emu-alg>
-        1. <ins>If _x_ is a Decimal128 value, then</ins>
-          1. <ins>Let _m_ be Decimal128ToDecimalString(_x_, *false*).</ins>
-          1. <ins>Let _e_ be the truncated exponent of _x_.</ins>
-          1. <ins>Let _n_ be the scaled significand of _x_.</ins>
-        1. <ins>Else,</ins>
-          1. <ins>Assert _x_ is a mathematival value.</ins>
-          1. Let _p_ be _maxPrecision_.
-          1. If _x_ = 0<ins> or _x_ is a zero Decimal128 object</ins>, then
-            1. Let _m_ be the String consisting of _p_ occurrences of the code unit 0x0030 (DIGIT ZERO).
-            1. Let _e_ be 0.
-            1. Let _xFinal_ be 0.
+        1. <ins>Let _m_ be *undefined*.</ins>
+        1. <ins>If _x_ is a Decimal128 value, set _m_ to Decimal128ToDecimalString(_x_, *false*).</ins>
+        1. Let _p_ be _maxPrecision_.
+        1. If _x_ = 0, then
```

Comment:
Could you reorganize this a bit to do:
1. Let _p_ be _maxPrecision_.
1. If _x_ is a Decimal128 value, let _m_ be...,
2. Else if _x_ = 0, then
3. Else,

So that we don't need to pre-initialize _m_ and we don't need to handle the case where it is undefined.

Reply:
done